install.sh

```diff
@@ -0,0 +1,59 @@
+
+#!/bin/bash
+
+set -euo pipefail
+
+echo "===== Cleaning previous install"
+rm -rf Release-iphoneos Release-iphonesimulator
+echo "===== Installing dependencies"
+pod install
+echo "===== Generating device variants"
+pushd Rome > /dev/null
+
+mkdir -p Release-iphoneos
+mkdir -p Release-iphonesimulator
+
+for d in *.framework/ ; do
+    frameworkName=`echo "${d}" | cut -d'/' -f 1`
+    libName=`echo "${frameworkName}" | cut -d'.' -f 1`
+    echo "-> ${frameworkName}"
+    cp -r ${frameworkName} Release-iphoneos/
+    mv ${frameworkName} Release-iphonesimulator/
+
+    pushd Release-iphoneos/${frameworkName} > /dev/null
+
+    mv ${libName} ${libName}.old
+    echo "   thinning for iOS"
+    lipo ${libName}.old -extract_family arm -extract_family arm64 -o ${libName}
+    rm ${libName}.old
+
+    dsym=../../../dSYM/iphoneos/${frameworkName}.dSYM
+    if [ -d ${dsym} ]; then
+        mv -f ${dsym} ../
+    fi
+
+    popd > /dev/null
+
+
+    pushd Release-iphonesimulator/${frameworkName} > /dev/null
```

Comment:
I'm aware this section is duplicated from above, but it serves its purpose. If a new architecture is to be split, we can possibly refactor this into something like `process(framework, forArchitecture, archiectures)` ... but I don't think it warrants a great deal of time...